image.yaml

```diff
@@ -29,9 +29,9 @@ modules:
           - name: amq-install
           - name: amq-chown
 artifacts:
-    - path: jboss-a-mq-6.3.0.redhat-329.zip
+    - path: jboss-a-mq-6.3.0.redhat-343.zip
       description: "Artifact is available on Customer Portal: https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=56371&product=jboss.amq.broker&version=6.3.0&downloadType=securityPatches"
```

Comment:
Please, update this to https://access.redhat.com/jbossnetwork/restricted/softwareDetail.html?softwareId=57981&product=jboss.amq.broker&version=6.3.0&downloadType=securityPatches